.travis.yml

```diff
@@ -31,9 +31,9 @@ before_script:
 
 script:
   - rustc --version
-  - if [ "$TRAVIS_RUST_VERSION" == "nightly" ]; then
-    rustup component add clippy;
-    cargo clippy --release;
-    fi
+  # - if [ "$TRAVIS_RUST_VERSION" == "nightly" ]; then
+  #   rustup component add clippy;
+  #   cargo clippy --release;
+  #   fi
```

Comment:
What's wrong with them?

Reply:
Look at travis logs, you'll see what dark magic is. :)

Reply:
It's installed by default now or not? There's no separate component for it.

Reply:
I couldn't make it work and at this point, it's just annoying.

Reply:
Ah no. The problem is: https://rust-lang-nursery.github.io/rust-toolstate/

Reply:
Ah indeed. I remove the last commit and then I merge.